dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -0,0 +1,679 @@
+/*
+    DSSCollection contains collection group & completion functionality for DSS.
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import NonFungibleToken from "./NonFungibleToken.cdc"
+import MetadataViews from 0xMETADATAVIEWSADDRESS
+
+// The DSSCollection contract
+//
+pub contract DSSCollection: NonFungibleToken {
+
+    // Contract Events
+    //
+    pub event ContractInitialized()
+
+    // NFT Collection Events
+    //
+    pub event Withdraw(id: UInt64, from: Address?)
+    pub event Deposit(id: UInt64, to: Address?)
+
+    // Events
+    //
+    pub event CollectionGroupCreated(
+        id: UInt64,
+        name: String,
+        description: String,
+        typeName: String,
+        startTime: UFix64?,
+        endTime: UFix64?,
+        timeBound: Bool
+    )
+    pub event CollectionGroupClosed(id: UInt64)
+    pub event ItemCreated(
+        id: UInt64,
+        itemID: UInt64,
+        points: UInt64,
+        itemType: String
+    )
+    pub event SlotCreated(
+        id: UInt64,
+        collectionGroupID: UInt64,
+        logicalOperator: String,
+        typeName: String
+    )
+    pub event ItemAddedToSlot(
+        id: UInt64,
+        slotID: UInt64,
+        collectionGroupID: UInt64
+    )
+    pub event CollectionNFTMinted(
+        id: UInt64,
+        collectionGroupID: UInt64,
+        serialNumber: UInt64,
+        completedBy: String,
+        completionDate: UFix64,
+        level: UInt8
+    )
+    pub event DSSCollectionNFTBurned(id: UInt64)
```

Comment:
nit: rename to `CollectionNFTBurned` to match the mint event

Reply:
Done